python/tk_desktop/desktop_window.py

```diff
@@ -818,10 +848,16 @@ def report_progress(percentage, message):
             # comes from.
             if most_recent_pipeline_configuration is None:
                 toolkit_manager.pipeline_configuration = None
+
+                # Enable user menu item to launch classic Project Setup wizard
```

Comment:
Here the comment should have been
`We've loaded this project before and saved its pipeline configuration id, so reload the same old one.`